src/wedge.jl

```diff
@@ -127,19 +127,45 @@ function embedding_unitary(partition, fockstates)
     return Diagonal(phases)
 end
 
+"""
+    fermionic_embedding(m, b, bnew)
+
+Compute the fermionic embedding of a matrix `m` in the basis `b` into the basis `bnew`.
+"""
+function fermionic_embedding(m, b, bnew)
+    # See eq. 20 in J. Phys. A: Math. Theor. 54 (2021) 393001
+    bbar_labs = setdiff([keys(bnew)...], [keys(b)...]) # arrays to keep order
+    qn = promote_symmetry(b.symmetry, bnew.symmetry)
+    bbar = FermionBasis(bbar_labs; qn)
+    return wedge([m, I], [b, bbar], bnew)
```

Comment:
```suggestion
    return wedge((m, I), (b, bbar), bnew)
```
These arrays are converted to Tuples later anyway, so that the wedge function is specialized on the number of matrices.